examples/formbot/tests/test_stories.yml

```diff
@@ -11,6 +11,7 @@ stories:
   - action: restaurant_form
```

Comment:
should this also get a `version: "2.0"`?

Reply:
Yep, it should. Just added!